addons/test_website/tests/test_reset_views.py

```diff
@@ -5,7 +5,7 @@
 from odoo.tools import mute_logger
 
 
-def break_view(view, fr='<p>placeholder</p>', to='<p t-field="not.exist"/>'):
+def break_view(view, fr='<p>placeholder</p>', to='<p t-field="no_record.exist"/>'):
```

Comment:
The new transformation system does not accept using python constants as variable name.